build.gradle

```diff
@@ -495,16 +494,16 @@ tasks.withType(Test).configureEach {
 }
 
 test {
-    useJUnit {
-        includeCategories 'bio.terra.common.category.Unit'
+    useJUnitPlatform {
```

Comment:
`useJUnit` enables JUnit 4 tests, whereas `useJUintPlatform` enables Junit 5 tests. In JUnit 5, the `tags` concept replaces the `categories` concept.